appinventor/blocklyeditor/src/blocks/text.js

```diff
@@ -55,8 +58,8 @@ Blockly.Blocks.text.setOutputOnFinishEdit = function(newValue) {
     this.outputConnection.setCheck(AI.BlockUtils.YailTypeToBlocklyType("text", AI.BlockUtils.OUTPUT));
   } else {
     // Remove the Number type from the output connection check if it exists.
-    const check = Array.from(AI.BlockUtils.YailTypeToBlocklyType("text", AI.BlockUtils.OUTPUT));
-    const numberIndex = check.indexOf('Number');
+    var check = Array.from(AI.BlockUtils.YailTypeToBlocklyType("text", AI.BlockUtils.OUTPUT));
```

Comment:
It seems like these could have been left as const?

Reply:
You've made me paranoid about using "modern" JavaScript features ;-). I'll change them back.